modin/pandas/accessor.py

```diff
@@ -35,7 +35,7 @@
 from modin.utils import _inherit_docstrings
 
 
-class BaseSparseAccessor(ClassLogger):
+class BaseSparseAccessor(ClassLogger, modin_layer="ACCESSOR"):
```

Comment:
```suggestion
class BaseSparseAccessor(ClassLogger):
```
This is also pandas API. Everthing under modin/pandas should be pandas API.